spec/yaml/swiftnav/sbp/bootload.yaml

```diff
@@ -22,15 +22,40 @@ include:
   - types.yaml
 definitions:
 
- - MSG_BOOTLOADER_HANDSHAKE:
+ - MSG_BOOTLOADER_HANDSHAKE_HOST:
+    id: 0x00B3
+    public: True
+    short_desc: Bootloading handshake request (host => device)
+    desc: |
+      The handshake message request from the host establishes a
+      handshake between the device bootloader and the host. The
+      response from the device is MSG_BOOTLOADER_HANDSHAKE_DEVICE.
+
+ - MSG_BOOTLOADER_HANDSHAKE_DEVICE:
     id: 0x00B0
     public: True
-    short_desc: Bootloading handshake (host <=> device)
+    short_desc: Bootloading handshake response (host <= device)
     desc: |
-      The handshake message establishes a handshake between the device
-      bootloader and the host.  The payload string contains the
-      bootloader version number, but returns an empty string for
-      earlier versions.
+      The handshake message response from the device establishes a
```

Comment:
Similarly, may also be feature creep, but having the response message be parseable:

```
short_desc: Bootloading handshake response (host <= device)
response: MSG_BOOTLOADER_HANDSHAKE_DEVICE
```